pom.xml

```diff
@@ -620,12 +635,12 @@
       <dependency>
         <groupId>org.reflections</groupId>
         <artifactId>reflections</artifactId>
-        <version>0.9.10</version>
+        <version>0.9.11</version>
```

Comment:
looks like the latest is actually 0.9.12, can we bump that to the latest version?